### PR TITLE
Adding multiple pretrained models

### PR DESCRIPTION
**Pull request by zhreshold**
| Model | Top 1 | Top 5 |
|:-----:|:------:|:------:|
| mobilenet0.25 | 0.5185 | 0.7608 |
| mobilenet0.5 | 0.6307 | 0.8475 |
| mobilenet0.75 | 0.6738 | 0.8782 |
| mobilenet1.0 | 0.7105 | 0.9006 |
| resnet50_v1 | 0.7540 | 0.9266 |
| resnet101_v1 | 0.7693 | 0.9334 |
| resnet152_v1 | 0.7727 | 0.9353 |
| resnet50_v2 | 0.7622 | 0.9297 |
| resnet101_v2 | 0.7747 | 0.9375 |
| resnet152_v2 | 0.7833 | 0.9409 |

Minor fixes:
- Fix resnetv1 strides, overrides #8414 if @ptrendx don't mind.
- Fixes batchnorm scale issue in mobilenet.

### CODE REVIEW

**Comment by chinakook** [comment]
Can you also share the hyper parameters how you train these networks?

**Comment by zhreshold** [comment]
@chinakook 
Checkout the defaults in https://github.com/zhreshold/mxnet/blob/model_zoo/example/gluon/train_imagenet.py
Note that it has vision branch merged already so you can't use master to run the training right now.

**Comment by zhreshold** [comment]
@szha ready
